Article about the Hitachi Meetup, please review. (#57)
techwriter-pl/techwriter-pl.github.io: blog/relacja-z-meetupu-hitachi-karate/index.md

---
title: 'Relacja z meetupu Hitachi Energy - edycja Karate'
authors: niewiarowski

> date: '2023-09-23'

Reviewer: @andrzejniewiaro , czy chcesz, żeby ten artykuł się pokazał z datą 23 września? Nie będzie go wtedy widać na stronie startowej, bo tam są już nowsze artykuły.

Author: good catch, zmienie na aktualna.

tags:
  - 'news'
  - 'z-kraju'
  - 'meetcontent'
  - 'spolecznosc'
coverImage: '1693820333064.png'
---

Jako ekipa techwriterów i techwriterek z [Hitachi Energy](https://www.linkedin.com/company/hitachienergy/mycompany/) w składzie: Julia Dąbrowska, Mateusz Barszcz, Piotr Sroka, Monika Dąbrowska, Andrzej Niewiarowski oraz członkini honorowa - Patrycja Pyrek - przeprowadziliśmy wyjątkowy meetup, którego spoiwem była kultura Japonii. 

<!--truncate-->

## Fusion Meetup o Technical Writingu i Karate

> ### Informacje o spotkaniu

Reviewer 2: Nie wiem czy  mamy jakieś zasady dotyczące nie zbijania nagłówków jeden po drugim? 
MS Style guide mówi:
![image](https://github.com/techwriter-pl/techwriter-pl.github.io/assets/119131560/99468d8f-edd4-4583-9029-29da731b0716)

Może usunąć nagłówek 'Informacje o spotkaniu' i z paragraf z linijki 22 zmienić we wstęp do sekcji Fusion Meetup o Technical Writingu i Karate?

Author: jest to dobry pomysl, dwa naglowki obok siebie nie sa potrzebne.

![](images/meta_prez.png)

> 12.09.2023 w siedzibie Hitachi Energy w Krakowie odbył się meetup dla pracowników firmy i gości z poza niej. Na spotkaniu stawiło się 40 osób. Piotr wziął na siebie dużą część obowiązków organizacyjnych, w czym ma już dużą wprawę, bo ma już na koncie przeprowadzenie writerskiego meetupu w 2019. Dla naszego zespołu ten fusion meetup to już drugie wspólne wydarzenie, bo w obecnym roku Anno Domini 2023 zorganizowaliśmy spotkanie z [Lance Cummings](https://www.linkedin.com/in/lance-cummings-phd/), który w krakowskiej siedzibie firmy opowiadał o tym, jak wykorzystywać *prompt engingeering* w arsenale pisarza technicznego.

Reviewer: ```suggestion
12.09.2023 w siedzibie Hitachi Energy w Krakowie odbył się meetup dla pracowników firmy i gości spoza niej. Na spotkaniu stawiło się 40 osób. Piotr wziął na siebie dużą część obowiązków organizacyjnych, w czym ma już dużą wprawę, bo ma już na koncie przeprowadzenie writerskiego meetupu w 2019. Dla naszego zespołu ten fusion meetup to już drugie wspólne wydarzenie, bo w obecnym roku Anno Domini 2023 zorganizowaliśmy spotkanie z [Lance Cummings](https://www.linkedin.com/in/lance-cummings-phd/), który w krakowskiej siedzibie firmy opowiadał o tym, jak wykorzystywać *prompt engingeering* w arsenale pisarza technicznego.
```

Reviewer: Jeżeli każdy akapit jest jedną linią, dodawanie komentarzy będzie zawsze dodaniem komentarza do całego akapitu. W Markdownie można stworzyć akapit pisząc po sobie wiele linii i oddzielamy go od następnego akapitu pusta linią.

W naszym repo mamy skonfigurowany Prettier. Jeżeli masz zainstalowany dodatek Prettier Code Formatter w VS Code, to możesz sformatować dokument naciskając Ctrl+S (Cmd+S na Macu). Wtedy każdy akapit podzieli się na linie o długości do 80 znaków i będzie łatwiej zrozumieć czego tyczy się dany komentarz.

Author: ok, zrobie tak nastepnym razem.

> 12.09.2023 w siedzibie Hitachi Energy w Krakowie odbył się meetup dla pracowników firmy i gości z poza niej. Na spotkaniu stawiło się 40 osób. Piotr wziął na siebie dużą część obowiązków organizacyjnych, w czym ma już dużą wprawę, bo ma już na koncie przeprowadzenie writerskiego meetupu w 2019. Dla naszego zespołu ten fusion meetup to już drugie wspólne wydarzenie, bo w obecnym roku Anno Domini 2023 zorganizowaliśmy spotkanie z [Lance Cummings](https://www.linkedin.com/in/lance-cummings-phd/), który w krakowskiej siedzibie firmy opowiadał o tym, jak wykorzystywać *prompt engingeering* w arsenale pisarza technicznego.

Reviewer 2: ```suggestion
12.09.2023 w siedzibie Hitachi Energy w Krakowie odbył się meetup dla pracowników firmy i gości spoza niej. Na spotkaniu stawiło się 40 osób. Piotr wziął na siebie dużą część obowiązków organizacyjnych, w czym ma dużą wprawę, bo ma już na koncie przeprowadzenie writerskiego meetupu w 2019. 

Dla naszego zespołu ten fusion meetup to już drugie wspólne wydarzenie, bo w 2023 zorganizowaliśmy spotkanie z [Lance Cummings](https://www.linkedin.com/in/lance-cummings-phd/), który w krakowskiej siedzibie firmy opowiadał o tym, jak wykorzystywać *prompt engineering* w arsenale pisarza technicznego.
```

Kilka stylistycznych uwag i mała literówka. Dodałabym też rozbicie na dwa paragrafy, żeby łątwiej się czytało :)

> 12.09.2023 w siedzibie Hitachi Energy w Krakowie odbył się meetup dla pracowników firmy i gości z poza niej. Na spotkaniu stawiło się 40 osób. Piotr wziął na siebie dużą część obowiązków organizacyjnych, w czym ma już dużą wprawę, bo ma już na koncie przeprowadzenie writerskiego meetupu w 2019. Dla naszego zespołu ten fusion meetup to już drugie wspólne wydarzenie, bo w obecnym roku Anno Domini 2023 zorganizowaliśmy spotkanie z [Lance Cummings](https://www.linkedin.com/in/lance-cummings-phd/), który w krakowskiej siedzibie firmy opowiadał o tym, jak wykorzystywać *prompt engingeering* w arsenale pisarza technicznego.

Reviewer 2: ```suggestion
12.09.2023 w siedzibie Hitachi Energy w Krakowie odbył się meetup dla pracowników firmy i gości spoza niej. Na spotkaniu stawiło się 40 osób. Piotr wziął na siebie dużą część obowiązków organizacyjnych, w czym ma już wprawę, bo ma na koncie przeprowadzenie writerskiego meetupu w 2019. 

Dla naszego zespołu ten fusion meetup to już drugie wspólne wydarzenie, bo w 2023 zorganizowaliśmy spotkanie z [Lance Cummings](https://www.linkedin.com/in/lance-cummings-phd/), który w krakowskiej siedzibie firmy opowiadał o tym, jak wykorzystywać *prompt engineering* w arsenale pisarza technicznego.
```

Kilka stylistycznych uwag i mała literówka. Dodałabym też rozbicie na dwa paragrafy, żeby łątwiej się czytało :)

### Dlaczego Japonia?

* Pracując w japońskiej firmie, jaką jest Hitachi Energy, mamy okazję obcować z kulturą japońską. 
* Wiele osób w zespole trenowało wschodnie sztuki walk i lubi oglądać filmy prezentujące mistrzów sztuk walk. 
* Japońska filozofia kaizen stanowi filar dla praktyk takich jak Lean Management, które stosowane są w Hitachi Energy. 
* W codziennej pracy technical writerów staramy się stale ulepszać metody pracy, co jest zgodne z japońską zasadą dążenia do doskonałości.
Niecodzienna fuzja tematów tego meetupu przypadła naszym gościom do gustu.

![](images/patrycja_jap.jpg)

> [Patrycja Pyrek](https://www.linkedin.com/in/patrycja-pyrek/), jako znawczyni tematu, wprowadziła nas we frapujące meandry kultury japońskiej, którą poznała z pierwszej ręki, kilkakrotnie odwiedzając Kraj Kwitnącej Wiśni i zgłębiając miejscowe zwyczaje. Słuchacze dowiedzieli się między innymi o tym, dlaczego nie należy wbijać pałeczek w ryż podczas posiłku oraz jak rozumieć szyld z godzinami otwarcia japońskiego sklepu od "23:00 - 25:00" :-)

Reviewer: ```suggestion
[Patrycja Pyrek](https://www.linkedin.com/in/patrycja-pyrek/), jako znawczyni tematu, wprowadziła nas we frapujące meandry kultury japońskiej, którą poznała z pierwszej ręki, kilkakrotnie odwiedzając Kraj Kwitnącej Wiśni i zgłębiając miejscowe zwyczaje. Słuchacze dowiedzieli się między innymi o tym, dlaczego nie należy wbijać pałeczek w ryż podczas posiłku oraz jak rozumieć szyld z godzinami otwarcia japońskiego sklepu od "23:00 - 25:00" 😊
```

> [Patrycja Pyrek](https://www.linkedin.com/in/patrycja-pyrek/), jako znawczyni tematu, wprowadziła nas we frapujące meandry kultury japońskiej, którą poznała z pierwszej ręki, kilkakrotnie odwiedzając Kraj Kwitnącej Wiśni i zgłębiając miejscowe zwyczaje. Słuchacze dowiedzieli się między innymi o tym, dlaczego nie należy wbijać pałeczek w ryż podczas posiłku oraz jak rozumieć szyld z godzinami otwarcia japońskiego sklepu od "23:00 - 25:00" :-)

Reviewer 2: A może napiszesz dlaczego japońskie sklepy są otwarte do 25:00? :D

Arigatō, Patrycja!

### Tech writer mistrzem wywierania wpływu

![](images/1694696569715.jpg)

W trakcie Meetupu [Monika Dąbrowska](https://www.linkedin.com/in/monika-klimek1/) opowiedziała o tym, co profesor Cialdini - specjalista na skalę światową w dziedzinie wywierania wpływu - może doradzić pisarzom technicznym, którzy mają problem z uzyskaniem uwag ekspertów domenowych co do opracowanego dokumentu. W codziennych bojach, często zwracamy uwagę na narzędzia, problemy techniczne i organizację pracy, a nie należy zapominać również o budowaniu zdrowych relacji ze współpracownikami. Monika przedstawiła nam perspektywę na mechanizmy rządzące budowaniem relacji między ludźmi. Otrzymaliśmy kilka praktycznych wskazówek, które mogą pomóc podczas codziennej pracy dokumentalisty. 

> Warto tu zaznaczyć, że choć tematyka prac Cialdiniego niektórym może przywodzić na myśl techniki makiawelicznej manipulacji, to znajomość uniwersalnych zasad, którymi rządzi się ludzka natura, pomaga nam lepiej rozumieć samych siebie i tę drugą osobę odgrywającą rolę *Subject Matter Expert*. Można spokojnie korzystać z tej wiedzy, mając krystalicznie czyste intencje i bez robienia kogokolwiek w bambuko :) 

Reviewer: Ta linia nie stworzy osobnego akapitu, bo brakuje pustej linii pomiędzy.

> Warto tu zaznaczyć, że choć tematyka prac Cialdiniego niektórym może przywodzić na myśl techniki makiawelicznej manipulacji, to znajomość uniwersalnych zasad, którymi rządzi się ludzka natura, pomaga nam lepiej rozumieć samych siebie i tę drugą osobę odgrywającą rolę *Subject Matter Expert*. Można spokojnie korzystać z tej wiedzy, mając krystalicznie czyste intencje i bez robienia kogokolwiek w bambuko :) 

Reviewer: ```suggestion
Warto tu zaznaczyć, że choć tematyka prac Cialdiniego niektórym może przywodzić na myśl techniki makiawelicznej manipulacji, to znajomość uniwersalnych zasad, którymi rządzi się ludzka natura, pomaga nam lepiej rozumieć samych siebie i tę drugą osobę odgrywającą rolę *Subject Matter Expert*. Można spokojnie korzystać z tej wiedzy, mając krystalicznie czyste intencje i bez robienia kogokolwiek w bambuko 😊
```

> Warto tu zaznaczyć, że choć tematyka prac Cialdiniego niektórym może przywodzić na myśl techniki makiawelicznej manipulacji, to znajomość uniwersalnych zasad, którymi rządzi się ludzka natura, pomaga nam lepiej rozumieć samych siebie i tę drugą osobę odgrywającą rolę *Subject Matter Expert*. Można spokojnie korzystać z tej wiedzy, mając krystalicznie czyste intencje i bez robienia kogokolwiek w bambuko :) 

Reviewer 2: ```suggestion
Warto tu zaznaczyć, że choć tematyka prac Cialdiniego niektórym może przywodzić na myśl techniki makiawelicznej manipulacji, to znajomość uniwersalnych zasad, którymi rządzi się ludzka natura, pomaga nam lepiej rozumieć samych siebie i tę drugą osobę odgrywającą rolę *SME*. Można spokojnie korzystać z tej wiedzy, mając krystalicznie czyste intencje i bez robienia kogokolwiek w bambuko :) 
```

Luźna uwaga o konieczności dodania 's jeśli użyjemy subject matter experta w zdaniu gdzie powinien się po polsku odmienić ;) 
Więc może lepiej go skrócić do SME?

Author: SME to staje sie wtedy dosc hermetyczne. staram sie pisac tez z mysla o osobach z zerowa wiedza o technical writingu. najzgrabniej przetlumaczyc na polski. Ekspert Domenowy? :D

### Wszystko ma swoje priorytety

![](images/andy_prez.jpg)

> Mimo opornego projektora, który przestał działać jak tylko do niego poszedłem (mam tę moc, mam tę moc!), udało mi się wyświetlić slajdy i opowiedzieć o zarządzaniu priorytetami projektów dokumentacyjnych. Podczas mojej blisko rocznej przygody w firmie Hitachi Energy miałem okazję przećwiczyć dostosowywanie priorytetów w dynamicznym środowisku, w którym tech writer ma więcej niż jeden projekt pod swoją pieczą. Cieszę się, że mogłem podzielić się obserwacjami, które mogą być pomocne również w kontekstach pozadokumentacyjnych, a także zaprezentowałem parę czerstwych memów własnego autorstwa.

Reviewer 2: ```suggestion
Mimo opornego projektora, który przestał działać jak tylko do niego podszedłem (mam tę moc, mam tę moc!), udało mi się wyświetlić slajdy i opowiedzieć o zarządzaniu priorytetami projektów dokumentacyjnych. Podczas mojej blisko rocznej przygody w firmie Hitachi Energy miałem okazję przećwiczyć dostosowywanie priorytetów w dynamicznym środowisku, w którym tech writer ma więcej niż jeden projekt pod swoją pieczą. Cieszę się, że mogłem podzielić się obserwacjami, które mogą być pomocne również w kontekstach pozadokumentacyjnych, a także zaprezentowałem parę czerstwych memów własnego autorstwa.
```

![](images/lee.png)

> Uczestnicy i uczestniczki zdarzenia okazywali zainteresowanie, zadali nawet pytania, a po zapaleniu świateł wszyscy byli obudzeni ;-) Dla mnie to definicja sukcesu.

Reviewer: ```suggestion
Uczestnicy i uczestniczki zdarzenia okazywali zainteresowanie, zadali nawet pytania, a po zapaleniu świateł wszyscy byli obudzeni 😉 Dla mnie to definicja sukcesu.
```

### Wejście Smoków

![](images/1694696568105.jpg)

> Wisienką na torcie była rozmowa z sensei Mają Wójcik i sensei Jakubem Piaseckim, którzy opowiadali:
> * o dążeniu do mistrzostwa w dziedzinie karate Oyama
> * o pozytywnych wartościach sportowych jakie niesie ze sobą karate
> * o tym jak znajdować w sobie wewnętrzną siłę, by systematycznie rozwijać swoje umiejętności
> * o tym, jak przekonać siebie samego do pójścia na pierwszy trening

Reviewer 2: ```suggestion
Wisienką na torcie była rozmowa z sensei Mają Wójcik i sensei Jakubem Piaseckim, którzy opowiadali dążeniu do mistrzostwa w dziedzinie karate Oyama oraz pozytywnych wartościach sportowych jakie niesie ze sobą ten sport.

Dowiedzieliśmy się również o tym jak:
* Znajdować w sobie wewnętrzną siłę, by systematycznie rozwijać swoje umiejętności
* Przekonać siebie samego do pójścia na pierwszy trening
```

Again, nie wiem czy mamy jakiś style guide, ale MS mówi o zaczynaniu bulleted lists z dużej litery.

Author: nie mamy style guide ale jak sie chce to mozna zaszczepiac rozne dobre praktyki. dzieki temu jak mi zwrocisz na cos uwage moge sie zastanowic czy moje podejscie domyslne aby jest najlepsze :)

> Mieliśmy przyjemność posłuchać opowieści młodych, ambitnych trenerów, a ich rozważania na temat samodoskonalenia i sposobów radzenia sobie z trudnościami mają również zastosowanie w kontekście technical writingu. Ważnym przesłaniem tamtego wieczora było to, że uprawianie sportu jest dobrą metodą na zagwarantowanie sobie właściwego stanu mentalnego. Aktywność fizyczna jest świetnym remedium na pracę umysłową w świecie IT: pozwala na zachowanie wewnętrznej równowagi i wydobycie pełni swojego potencjału.

Reviewer 2: ```suggestion
Mieliśmy przyjemność posłuchać opowieści młodych, ambitnych trenerów, a ich rozważania na temat samodoskonalenia i sposobów radzenia sobie z trudnościami mają również zastosowanie w kontekście technical writingu. Ważnym przesłaniem tamtego wieczoru było to, że uprawianie sportu jest dobrą metodą na zagwarantowanie sobie właściwego stanu mentalnego. Aktywność fizyczna jest świetnym remedium na pracę umysłową w świecie IT: pozwala na zachowanie wewnętrznej równowagi i wydobycie pełni swojego potencjału.
```

![](images/sensei.jpg)

Jakub wspomniał również o swojej drugiej pasji życiowej, którą jest fotografia. Karate było motywem przewodnim jego pracy dyplomowej, a przykłady jego prac mogliśmy podziwiać w trakcie spotkania.

![](images/interview_karate2.png)

Podziękowania dla [Piotra Sroki](https://www.linkedin.com/in/piotrsrokatechwriter/) i [Mateusza Barszcza](https://www.linkedin.com/in/mattbarszcz/), którzy przygotowali pytania i wcielili się role moderatorów spotkania.

### Po prezentacjach
![](images/team_he.png)
Po długim i pełnym wrażeń wieczorze, gospodarze i goście mogli poczęstować się przekąskami, powymieniać się uwagami i poznać nowych ludzi. Każdy mógł zabrać do domu wybrany gadżet z logo Hitachi Energy. W tym miejscu podziękowania dla [Sandry Cieniawskiej](https://www.linkedin.com/in/sandracieniawska/) i [Klaudii Sucharzewskiej](https://www.linkedin.com/in/klaudia-sucharzewska-048ba721a/), która pomogły w zorganizowaniu kateringu i gadżetów.

> Słyszałem, że uczestnicy chwalili sobie to wydarzenie i odbiło się ono w firmie pewnym echem :)

Reviewer: ```suggestion
Słyszałem, że uczestnicy chwalili sobie to wydarzenie i odbiło się ono w firmie pewnym echem 😊
```

> Słyszałem, że uczestnicy chwalili sobie to wydarzenie i odbiło się ono w firmie pewnym echem :)

Reviewer: "odbiło się ono w firmie pewnym echem" - mam wrażenie, że to zdanie to otwarcie jakiejś anegdoty, ale brakuje tej anegdoty. Może chcesz napisać dlaczego masz takie wrażenie, na przykład ludzie cytowali fakty o Japonii przy kawie?

Author: nie chcialem zeby brzmialo to zbyt bunczucznie, np.: "to zdarzenie bylo na ustach wszystkich w firmie", kojarze jakies komentsy na intranecie i tez zaslyszane pozytywne opinie ludzi pozadokumentacyjnych. Nie brzmi to jak wyborna angedota :)
nie wiem jeszcze czym to zastapic, pomysle.

Reviewer: Może też zostać jak jest, nie jest wcale źle 💜
